tests/test_embeddings.py

```
@@ -92,7 +99,16 @@ async def test_encode_batch_chunks_requests_and_normalises_dimensions() -> None:
 
     assert len(result.vectors) == len(payloads)
     assert all(len(vector) == 3 for vector in result.vectors)
-    assert manager.calls == [payloads[:2], payloads[2:4], payloads[4:]]
+    expected_batches = [payloads[:2], payloads[2:4], payloads[4:]]
+    assert len(manager.calls) == len(expected_batches)
+    for recorded_batch, expected_texts in zip(manager.calls, expected_batches):
```

Ruff (0.14.0)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)